spec/fixtures/cassettes/features/spree/admin/refund.yml

```diff
@@ -0,0 +1,572 @@
+        YORK CITY","country":"US","county":"QUEENS","state":"NY"},"order_total_amount":40.0,"rate":0.08875,"shipping":0.0,"tax_source":"destination","taxable_amount":40.0}}'
+    http_version:
+  recorded_at: Tue, 23 Jan 2024 20:27:41 GMT
+- request:
+    method: post
+    uri: https://api.taxjar.com/v2/transactions/orders
+    body:
+      encoding: ASCII-8BIT
+      string: '{"customer_id":"1","to_country":"US","to_zip":"11430","to_city":"Herndon","to_state":"NY","to_street":"A
+        Different Road","line_items":[{"id":2,"quantity":1,"product_identifier":"SKU-2","description":"Second
+        Product - Master","product_tax_code":"TaxCode - 597842","unit_price":"10.0","discount":0,"sales_tax":"0.89"},{"id":3,"quantity":2,"product_identifier":"SKU-3","description":"Third
+        Product - Master","product_tax_code":"TaxCode - 597842","unit_price":"10.0","discount":0,"sales_tax":"1.78"}],"transaction_id":"R525233498","transaction_date":"2024-01-23T20:27:39Z","amount":"40.0","shipping":"0.0","sales_tax":"3.56"}'
```

Comment:
It looks like we're not accounting for the reduced quantity in the `amount` field on the transaction here. This could be related to the fact that the event we hook into is fired before `persist_totals` is called, and thus giving us a false positive failure in the specs.